examples/app/src/App.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Checkbox, Box } from "@strapi/design-system";
+import { Checkbox as BuffetCheckbox } from "@buffetjs/core";
+import "./App.css";
+
+function App() {
+  const [val, setValue] = useState();
```

Comment:
It's a nice start, but what do you think later to create a more dynamic form so it is easier to test the other input components?

Reply:
Yes, I will do it when we will have more components. Not sure it is relevant to do it actually with a checkbox and radio only